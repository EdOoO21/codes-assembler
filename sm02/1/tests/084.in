-420 871 587 -924 1 -992 68 354 -348 706 -598 -58 21 749 129 544 514 -249 246 -693 115 939 548 468 -465 76 -43 -184 824 833 486 -324 235 -551 -743 352 922 -31 -911 50 660 -910 -695 -905 23 82 407 -403 97 -780 -976 -506 -586 851 -135 -723 -401 441 -770 -444 -753 856 497 -791 114 -850 -48 130 -940 -701 560 587 600 490 451 -15 -949 457 235 288 335 -723 805 123 219 261 895 -611 -678 -459 -621 -348 802
