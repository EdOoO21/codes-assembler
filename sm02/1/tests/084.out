802
-348
-621
-459
-678
-611
895
261
219
123
805
-723
335
288
235
457
-949
-15
451
490
600
587
560
-701
-940
130
-48
-850
114
-791
497
856
-753
-444
-770
441
-401
-723
-135
851
-586
-506
-976
-780
97
-403
407
82
23
-905
-695
-910
660
50
-911
-31
922
352
-743
-551
235
-324
486
833
824
-184
-43
76
-465
468
548
939
115
-693
246
-249
514
544
129
749
21
-58
-598
706
-348
354
68
-992
1
-924
587
871
-420
